zaza/charm_lifecycle/func_test_runner.py

```diff
@@ -187,6 +187,7 @@ def func_test_runner(keep_model=False, smoke=False, dev=False, bundles=None,
     """
```

Comment:
Probably need some docs somewhere (here in the docstring?) to say that it does attempt to guess the bundle iff there is only one scoped alias?

Reply:
will do, wanted to get another set of eyes on it early :)